src/tools/api.js

```diff
@@ -16,53 +17,11 @@ const { API_KEY, UUID } = process.env;
  * }} param0
  */
 async function callForSearch({ query, client }) {
+    await Apify.setValue('searchQuery', SearchQuery);
     const response = await client({
         url: '/batched',
-        payload: [{
-            query: SearchQuery,
-            variables: {
-                request: {
-                    query,
-                    limit: 1,
-                    scope: 'WORLDWIDE',
-                    locale: 'en-US',
-                    scopeGeoId: 1,
-                    searchCenter: null,
-                    types: [
-                        'LOCATION',
-                        'QUERY_SUGGESTION',
-                        'LIST_RESULT',
-                    ],
-                    locationTypes: [
-                        'GEO',
-                        'AIRPORT',
-                        'ACCOMMODATION',
-                        'ATTRACTION',
-                        'ATTRACTION_PRODUCT',
-                        'EATERY',
-                        'NEIGHBORHOOD',
-                        'AIRLINE',
-                        'SHOPPING',
-                        'UNIVERSITY',
-                        'GENERAL_HOSPITAL',
-                        'PORT',
-                        'FERRY',
-                        'CORPORATION',
-                        'VACATION_RENTAL',
-                        'SHIP',
-                        'CRUISE_LINE',
-                        'CAR_RENTAL_OFFICE',
-                    ],
-                    userId: null,
-                    context: {
-                        typeaheadId: Date.now(),
-                        uiOrigin: 'SINGLE_SEARCH_HERO',
-                    },
-                    articleCategories: [],
-                    enabledFeatures: ['typeahead-q'],
-                },
-            },
-        }],
+        body: [{ query: SearchQuery,
+            variables: { request: { query, limit: 10, scope: 'WORLDWIDE', locale: 'en-US', scopeGeoId: 1, searchCenter: null, types: ['LOCATION', 'QUERY_SUGGESTION', 'USER_PROFILE', 'RESCUE_RESULT'], locationTypes: ['GEO', 'AIRPORT', 'ACCOMMODATION', 'ATTRACTION', 'ATTRACTION_PRODUCT', 'EATERY', 'NEIGHBORHOOD', 'AIRLINE', 'SHOPPING', 'UNIVERSITY', 'GENERAL_HOSPITAL', 'PORT', 'FERRY', 'CORPORATION', 'VACATION_RENTAL', 'SHIP', 'CRUISE_LINE', 'CAR_RENTAL_OFFICE'], userId: null, context: { searchSessionId: '91CDC537DA7536533B97CF5EEFE80DA41638306855528ssid', typeaheadId: '1638306877219', uiOrigin: 'SINGLE_SEARCH_HERO', routeUid: '20b035aa-a83c-4750-ad15-234206bd6c5d' }, articleCategories: ['default', 'love_your_local', 'insurance_lander'], enabledFeatures: ['typeahead-q'] } } }],
```

Comment:
Maybe just put this to separate file so it doesn't create this super long row. It is a constant anyway.

Reply:
"query" is dynamic there.
I will format it like a normal object.
Looks good for me in this way)
Or shall I still make a separate file for it?

Reply:
Looks good now